main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -118,4 +158,32 @@ object Completion {
     */
   case class WithCompletion(name: String, priority: String, textEdit: TextEdit, documentation: Option[String],
                             insertTextFormat: InsertTextFormat) extends Completion
+
+
+  /**
+    * Represents an importNew completion (java constructors)
+    *
+    * @param constructor      the constructor.
+    * @param clazz            clazz is the clazz in string form.
+    * @param aliasSuggestion  an alias for the function.
+    */
+  case class ImportNewCompletion(constructor: Constructor[_], clazz: String, aliasSuggestion: Option[String], context: CompletionContext) extends Completion
+
+  /**
+    * Represents an importMethod completion (java methods)
+    *
+    * @param method  the method.
+    * @param clazz   clazz is the clazz in string form.
+    */
+  case class ImportMethodCompletion(method: Method, clazz: String, context: CompletionContext) extends Completion
```

Comment:
What's the class? Could it not also be the object? e.g. java.lang.reflect.Class?

Reply:
I don't know how to resolve this, since the clazz comes from the regex and I'm not quite sure how that works or what it does..

If we take the java.lang.reflect.Class object, and call its method `getName()`, I'm not sure that we get the desired string that we need as `label` in `CompletionItem`.

Reply:
I see okay.